### ISSUE TITLE
Where can I download SMRT Analysis 3.0.x?

### ISSUE THREAD

**Issue by pb-jlandolin** [PacificBiosciences/SMRT-Link#10]
SMRT Analysis 3.0.x is only available as a module within SMRT Link 1.0.x. You must perform a fresh install of SMRT Link 1.0.x in order to access SMRT Analysis 3.0.x.

SMRT Link will be available for download at the PacBio website http://www.pacb.com/support/software-downloads/.

**Comment by afif-elghraoui** [comment]
Is this supposed to become available sometime soon? I see many issues reported by users, but I don't know how they're getting the software. Is it only available so far to people who bought the Sequel?

**Comment by jkaralius** [comment]
SMRT Link is currently a limited release, available only to Sequel customers. 
We anticipate full release late Summer 2016.

**Comment by pb-jlandolin** [comment]
Yes, those issues are reported by folks who have purchased a PacBio Sequel.

**Comment by yech1990** [comment]
It is late 2016 now

**Comment by kbessonov1984** [comment]
There are bits and pieces of the SMRTLink suite software all over the PacBio GitHub repository. As intermediate solution it is possible to install pythonic "pbsmrtpipe" library from this repo and run all 61 pipelines via a command line ... SMRTflow seems to incorporate SMRTLink server but not fully ... It is a shame that SMRTLink incorporating all tools, is still not publically accessible ...

**Comment by WenchaoLin** [comment]
Baidu share download link: [http://www.ngsgo.com/softwares/523.html](http://www.ngsgo.com/softwares/523.html )
`SMRT Link 4.0.0.190159`

**Comment by jkaralius** [comment]
SMRT Link is available for download at the PacBio website http://www.pacb.com/support/software-downloads/

**Comment by kbessonov1984** [comment]
In addition, to the newest version on the public website, other vesions could be located at https://downloads.pacbcloud.com/public/software/installers/ 
Unzip passwords for versions 4.0 and 5.0.1 are 'SmrT3chN' and 'nX4dmaR%'

**Comment by moskalenko** [comment]
5.0.1 does not support more recent pacbio chemistry. What's the password for the smrtlink_5.1.0.21036.zip already available on the download site?

**Comment by npavlovikj** [comment]
@moskalenko , @kbessonov1984 , were you able to find the password for "smrtlink 5.1.0"? I am having the same issue...

**Comment by moskalenko** [comment]
No, I was told that it was a beta version only distributed to select customers. I was provided with a chemistry bundle to update our existing 5.0.1 install.

**Comment by npavlovikj** [comment]
Thank you so much @moskalenko, I appreciate your reply! I emailed their support, so hopefully I can get the chemistry bundle as well.

**Comment by WenchaoLin** [comment]
@moskalenko You can download SMRTLINK Chemistry Bundle v5.0.1 from https://www.ngsgo.com/archives/2767.html .
